src/diffusers/utils/outputs.py

```diff
@@ -101,6 +101,13 @@ def __setitem__(self, key, value):
         # Don't call self.__setattr__ to avoid recursion errors
         super().__setattr__(key, value)
 
+    def __reduce__(self):
```

Comment:
That works! In Python, the `__reduce__` function is more or less exclusively reserved for pickle no? So there should be no other side-effects that could be trigger here?